plugins/woocommerce/src/Admin/Features/ProductBlockEditor/BlockRegistry.php

```diff
@@ -22,7 +22,6 @@ class BlockRegistry {
 	const PRODUCT_BLOCKS = [
 		'woocommerce/conditional',
 		'woocommerce/product-catalog-visibility-field',
-		'woocommerce/product-category-field',
```

Comment:
Should we add an entry for `woocommerce/taxonomy-field`?

I'm not really sure why it works without it, or what any problem would be.